### PR TITLE
Revise CONTRIBUTIONS to dictate requirements for Pull Requests, etc.

### PR DESCRIPTION
**Pull request by teward**
Due to a small flurry of less-than-useful pull requests with no justification, etc. which in turn require us to go *digging* to actually understand what the pull reqs do, we should more explicitly define our requirements for pull requests including Minimum Requisite Information.

watchlist and blacklist changes are much simpler to review but when we start putting in heavier code changes, functionality, etc. it becomes more necessary to have more details and justifications in the pull request.

This Pull Request is a draft until we get review of language, etc. at which point we'll make it a standard Pull Request.

Anyone with commit access is allowed to revise the branch, which is why I made it under SmokeDetector.

### CODE REVIEW

File: CONTRIBUTING_PULLREQS.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+### Requirements when making Code Changes (NOT watchlist/blacklist changes)
+
+When you are making pull requests to the code base for SmokeDetector, the
+Charcoal team needs he pull request to contain enough information for us to
```
Comment (ArtOfCode-) [inline CONTRIBUTING_PULLREQS.md:4]: `he` => `the`

File: CONTRIBUTING_PULLREQS.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+ - What does your pull request change or introduce to the project?
+ - What is the justification for the inclusion of your Pull Request (or, what 
+   problem does it solve?)
+ - What does your code (if new) actually do?  Summarize what your new code 
+   does (either within the code via comments or with your Pull Request)
+   
+Pull requests that do not have ample information attached to them for 
+justification, etc. may be rejected or may be put on hold until we get more 
+information. Failure to provide ample information must
```
Comment (ArtOfCode-) [inline CONTRIBUTING_PULLREQS.md:18]: must?

Comment (teward) [reply CONTRIBUTING_PULLREQS.md:18]: Force of habit to write this, we can revise this easy.

File: CONTRIBUTING_PULLREQS.md (hunk 1 above)
Comment (teward) [reply CONTRIBUTING_PULLREQS.md:4]: Fast typing, heh.  Thanks.

File: CONTRIBUTING_PULLREQS.md (hunk 2 above)
Comment (ArtOfCode-) [reply CONTRIBUTING_PULLREQS.md:18]: Must what, though?

Comment (teward) [reply CONTRIBUTING_PULLREQS.md]: gdi autocorrect.  This didn't copy and then when i retyped failed things.  Altered.

**Comment by ArtOfCode-** [comment]
:+1:

**Comment by teward** [comment]
Nobody merge this until its sat for a while.  I'd say 48h in case anyone else wants to make revisions.

At that time we can accept this if all checks out.

**Comment by teward** [comment]
> Nobody merge this until its sat for a while. I'd say 48h in case anyone else wants to make revisions.
> 
> At that time we can accept this if all checks out.

With no further complaints or changes, I am going to merge this.

**Pull request merged**